src/services/webPlatformUtils.service.ts

```diff
@@ -24,7 +24,7 @@ export class WebPlatformUtilsService implements PlatformUtilsService {
             this.browserCache = DeviceType.FirefoxBrowser;
         } else if (navigator.userAgent.indexOf(' OPR/') >= 0) {
             this.browserCache = DeviceType.OperaBrowser;
-        } else if (navigator.userAgent.indexOf(' Edge/') !== -1) {
+        } else if (navigator.userAgent.indexOf(' Edge/') !== -1 || navigator.userAgent.indexOf(' Edg/') !== -1) {
```

Comment:
By doing this, you will introduce a bunch of hacks that are in place for Edge device type. We need to strip those out since we will no longer be deploying for old edge.